### PR TITLE
Feature: Implement Lazy Loading in explore page

### PR DESCRIPTION
**Pull request by AndreMigz**
##  Description:
- This changes aims to implement lazy loading on the explore page.

## Demo: 
[Screencast from 2024-08-30 17-32-20.webm](https://github.com/user-attachments/assets/eb9aa924-6611-49dd-aff8-2cc610f8ba79)

### CODE REVIEW

File: app/views/notes/explore.js.erb [hunk 1]
```diff
@@ -0,0 +1,8 @@
+$('#notes-div').append('<%= j render @notes %>');
+<% if @notes.next_page %>
+  $('.pagination').replaceWith('<%= j will_paginate @notes %>');
+<% else %>
+  $(window).off('scroll');
+  $('.pagination').remove();
+<% end %>
+$('#loader').addClass('hidden');
```
Comment (iprog21) [inline app/views/notes/explore.js.erb]: sir @AndreMigz . To remove this "minus" sign. add empty newline at the bottom. Github is following POSIX standard.

**Review by iprog21 — CHANGES_REQUESTED** [review]
minor fixes only @AndreMigz

File: app/views/notes/explore.js.erb (hunk 1 above)
Comment (AndreMigz) [reply app/views/notes/explore.js.erb]: Hi @iprog21 I already committed the suggested changes :rocket:

**Review by iprog21 — APPROVED** [review]
all goods

**Pull request merged**